plotly/tools.py

```diff
@@ -6208,6 +6207,9 @@ def create_dendrogram(X, orientation="bottom", labels=None,
         if len(s) != 2:
             exceptions.PlotlyError("X should be 2-dimensional array.")
 
+        if distfun is None:
+            distfun = scs.distance.pdist
+
```

Comment:
Right. It's just that the error won't be "scipy is not defined" because it won't break on the `import scipy` step, it'll break on the `scs.distance.pdist` step which means that the error will probably be `scs is not defined` which isn't very intuitive.

Could you try creating a clean virtualenv without scipy and try calling this function and seeing what the error message is? Scipy may be common but Plotly may be the first package and experience that users have in Python and we want to make it as easy as possible for people to get going. We can fix the issue in a separate PR but it'd be nice to know how this will break for our users.